{"name": "Проверка ошибки при авторизации с неправильным логином или паролем", "status": "passed", "description": "Проверяется ошибка при вводе неправильного логина или пароля.", "parameters": [{"name": "login_password", "value": "{'login': 'Friend.317', 'password': 'wrongpassword'}"}], "start": 1723375731840, "stop": 1723375731971, "uuid": "c9a9d523-9f22-4575-b7a6-d7abe0cab727", "historyId": "712a969f3ca9f589a65c63e87b344cc1", "testCaseId": "432e0cc92bda3737e4e249e8e3172c47", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_invalid_credentials", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "HOME-PC"}, {"name": "thread", "value": "6908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}